specification/source/cdl.rst

```diff
@@ -1688,118 +1688,75 @@ Therefore, tools that process CDL can infer the following information:
+                    metadataLanguage="Text" "This is a temperature reading input that should be hardwired to heating coil temperature sensor"));
 
-For Haystack, the ``__cdl_annotation`` is
+* Depending on the metadataLanguage and the format, the metadata could vary from an RDF subgraph to simple plain text. The text ``cdl_instance_name`` within the metadata can be used for referring to the current element if it has the same name as the CDL element. In the above example, the tool that parses the semantic information will infer the Brick metadata as ``bldg:THeaCoiSup_in a Brick:Hot_Water_Supply_Temperature_Sensor .`` and the Project Haystack identifier as ``{"@id": "THeaCoiSup_in"}``. This will avoid the user having to repeat the name of the element and makes it less prone to errors and inconsistencies.
 
-.. code-block:: modelica
+* The metadata defined in the top-level bl annotations will be accessible to elements (inputs, outputs etc.) defined within the sequence. Additionally, semantic information attached to an element can refer to elements that were defined earlier in the sequence. This way, the semantic information attached to an element can refer to another element within the sequence that was defined previously.
 
-   haystack_annotation:
-      haystack "(" JSON ")"
-
-where ``JSON`` is the JSON encoding of the Haystack object.
-
-[For example, the AHU discharge air temperature setpoint
-is in Haystack declared as
-
-.. code::
-
-   id: @whitehouse.ahu3.dat
-   dis: "White House AHU-3 DischargeAirTemp"
-   point
-   siteRef: @whitehouse
-   equipRef: @whitehouse.ahu3
-   discharge
-   air
-   temp
-   sensor
-   kind: "Number"
-   unit: "degF"
-
-can be declared in CDL as
-
-.. code::
-
-   annotation(__cdl( haystack=
-     "{\"id\"   : \"@whitehouse.ahu3.dat\",
-       \"dis\" : \"White House AHU-3 DischargeAirTemp\",
-       \"point\" : \"m:\",
-       \"siteRef\" : \"@whitehouse\",
-       \"equipRef\" : \"@whitehouse.ahu3\",
-       \"discharge\" : \"m:\",
-       \"air\"       : \"m:\",
-       \"temp\"      : \"m:\",
-       \"sensor\"    : \"m:\",
-       \"kind\"       : \"Number\"
-       \"unit\"       : \"degF\"}"));
-
-
-Tools that process CDL can interpret the ``brick`` or ``haystack`` annotation,
-but for control purposes CDL will ignore it. [This avoids potential conflict for entities that
-are declared differently in Brick (or Haystack) and CDL, and may be conflicting.
-For example, the above sensor input declares in Haystack that it belongs
-to an ahu3. CDL, however, has a different syntax to declare such dependencies:
-In CDL, through the ``connect(whitehouse.ahu3.TSup, ...)`` statement,
-a tool can infer what upstream component sends the input signal.]
+* Tools that process CDL can interpret the ``__semantic`` annotation, but for control purposes CDL will ignore it. [This avoids potential conflict for entities that are declared differently across these metadata standards and CDL, and may be conflicting.]
```

Comment:
Above that line is an (old) paragraph
```
The metadata defined in the top-level bl annotations will be accessible to elements (inputs, outputs etc.) defined within the sequence. Additionally, semantic information attached to an element can refer to elements that were defined earlier in the sequence. This way, the semantic information attached to an element can refer to another element within the sequence that was defined previously.
```
Please rewrite this and use instance rather than element when you mean an instance of an input, output, block etc. as we do not have a term called "element" in the CDL description. (We should always use the precise technical terms to avoid confusing the new reader.)